Northstar.Client/mod/resource/northstar_client_localisation_english.txt

```diff
@@ -304,6 +304,7 @@ Press Yes if you agree to this. This choice can be changed in the mods menu at a
 		"SHOW_ONLY_DISABLED" "Only Disabled"
 		"SHOW_ONLY_NOT_REQUIRED" "Only Optional Mods"
 		"SHOW_ONLY_REQUIRED" "Only Required Mods"
+		"MOD_REQUIRED_WARNING" " : This mod may get (un)loaded when joining a server"
```

Comment:
Formatting shouldn't be part of the localisation.

Although I'm not sure if sticking
```
labelText "#MOD_REQUIRED_WARNING"
```
actually gets it localised.

Reply:
Looking at #673 we could probably just drop the formatting entirely.

Reply:
> Although I'm not sure if sticking
> 
> ```
> labelText " : #MOD_REQUIRED_WARNING"
> ```
> 
> actually gets it localised properly.

If you want to do something like that you'll have to set the text in script to prepend the ` : ` (which is dumb)